operational/compute_instance_migration/risc-compute-instance-migration-recos.pt

```diff
@@ -321,5 +321,8 @@ EOS
 end
 
```

Comment:
Is this policy supports new UI, as this policy is constructing the incident table dynamically in detail_template.